src/jit/hwintrinsiccodegenxarch.cpp

```diff
@@ -428,6 +472,56 @@ void CodeGen::genHWIntrinsic_R_R_RM_I(GenTreeHWIntrinsic* node, instruction ins)
     }
 }
 
+template <typename HWIntrinsicSwitchCaseBody>
+void CodeGen::genJumpTableFallback(regNumber                 nonConstImmReg,
+                                   regNumber                 baseReg,
+                                   regNumber                 offsReg,
+                                   HWIntrinsicSwitchCaseBody emitSwCase)
+{
+    assert(nonConstImmReg != REG_NA);
```

Comment:
A comment about how/where inappropriate ranges are handled would be useful.

Reply:
Added the comment in the function header.